ocsci/pytest_customization/ocscilib.py

```diff
@@ -42,6 +42,13 @@ def pytest_addoption(parser):
         dest='cluster_name',
         help="Name of cluster",
     )
+    parser.addoption(
+        '--skip-teardown',
+        dest='skip_teardown',
+        action="store_true",
+        default=False,
```

Comment:
I would prefer True as default, Cluster setup is time consuming.

Reply:
I see that point, but I think we should change the flag to be ``--teardown`` and default it to ``False`` in that case though. Thoughts?

Reply:
Agree on the flag switch, thanks!